modules/medusa.yml

```diff
@@ -0,0 +1,18 @@
+name: medusa
+description: Easily connect to Medusa from your Nuxt 3 application
+repo: Baroshem/nuxt-medusa
+npm: nuxt-medusa
+icon: medusa.svg
+github: https://github.com/Baroshem/nuxt-medusa/
+website: https://nuxt-medusa.vercel.app/
+learn_more: https://nuxt-medusa.vercel.app/
+category: CMS
```

Comment:
suggestion: Maybe we can create a new filter for e-commerce? I am sure there is a module for Shopify and Swell :)

Reply:
Yes I was already strugglen which category Medusa is. Adding an e-commerce Category would be nice 👯

Reply:
@Baroshem https://github.com/nuxt/modules/blob/main/lib/categories.ts#L10 There is actually an E-Commerce Category